src/iexamine.cpp

```diff
@@ -3530,45 +3584,44 @@ void iexamine::autodoc( player &p, const tripoint &examp )
             const itype *itemtype = it->type;
             const bionic_id &bid = itemtype->bionic->id;
 
-            if( p.has_bionic( bid ) ) {
-                popup( _( "You have already installed this bionic."  ) );
+            //~ %1$s is installer name and shouldn't need translation, %2$s is bionic CBM display name, %3$s is patient name and shouldn't need translation
+            add_msg( _( "%1$s prepares to install the %2$s on %3$s." ), installer.name,
+                     it->display_name().c_str(), patient.name );
+            if( patient.has_bionic( bid ) ) {
+                //~ %1$s is the You have/<npcname> has construct described above
```

Comment:
Translators on Transifex won't understand what sort of "construct described above" is meant here.

Reply:
I am learning I do not understand the translation support system. will fix.